source/docs/software/vscode-overview/creating-robot-program.rst

```diff
@@ -86,6 +86,22 @@ The ``Romi - Command Bot`` template provides a ``RomiDrivetrain`` subsystem that
 
 This subsystem also provides functions for retrieving and resetting the Romi's onboard encoders.
 
+### XRP
+
+Teams using a :ref:`XRP <docs/xrp-robot/index:Getting Started with XRP>` should use the ``XRP - Timed`` or ``XRP - Command Bot`` template.
+
+#### XRP - Timed
+
+The ``XRP - Timed`` template provides a ``XRPDrivetrain`` class that exposes an ``arcadeDrive(double xaxisSpeed, double zaxisRotate)`` method. It's up to the user to feed this arcadeDrive function.
```

Comment:
We should be exact with the naming here.  It looks like they are called ``XRP - Command Robot`` and ``XRP - Timed Robot``.  This is also a problem with the Romi section.  I realize you didn't write that but I think improving it the same way would help!